src/agent/handlers/clientManifest.ts

```diff
@@ -0,0 +1,94 @@
+import type { AgentRPCRequestParams, AgentRPCResponseResult } from '../types';
+import type { EchoMessage } from './types';
+import type {
+  AgentClaimMessage,
+  ClaimIdMessage,
+  GitPackMessage,
+  HolePunchRelayMessage,
+  NodeAddressMessage,
+  NodeIdMessage,
+  SignedNotificationEncoded,
+  VaultInfo,
+  VaultsGitInfoGetMessage,
+  VaultsGitPackGetMessage,
+  VaultsScanMessage,
+} from './types';
+import { DuplexCaller, ServerCaller, UnaryCaller } from '../../rpc/callers';
+
+const echo = new UnaryCaller<
+  AgentRPCRequestParams<EchoMessage>,
+  AgentRPCResponseResult<EchoMessage>
+>();
+
+const nodesChainDataGet = new ServerCaller<
+  AgentRPCRequestParams<ClaimIdMessage>,
+  AgentRPCResponseResult<AgentClaimMessage>
+>();
+
+// Const nodesClaimsGet = new UnaryCaller<
+//   AgentRPCRequestParams,
+//   AgentRPCResponseResult
+// >();
+
+const nodesClosestLocalNodesGet = new ServerCaller<
+  AgentRPCRequestParams<NodeIdMessage>,
+  AgentRPCResponseResult<NodeAddressMessage>
+>();
+
+const nodesCrossSignClaim = new DuplexCaller<
+  AgentRPCRequestParams<AgentClaimMessage>,
+  AgentRPCResponseResult<AgentClaimMessage>
+>();
+
+const nodesHolePunchMessageSend = new UnaryCaller<
+  AgentRPCRequestParams<HolePunchRelayMessage>,
+  AgentRPCResponseResult
+>();
+
+const notificationsSend = new UnaryCaller<
+  AgentRPCRequestParams<SignedNotificationEncoded>,
+  AgentRPCResponseResult
+>();
+
+const vaultsGitInfoGet = new ServerCaller<
+  AgentRPCRequestParams<VaultsGitInfoGetMessage>,
+  AgentRPCResponseResult<VaultInfo | GitPackMessage>
+>();
+
+const vaultsGitPackGet = new ServerCaller<
+  AgentRPCRequestParams<VaultsGitPackGetMessage>,
+  AgentRPCResponseResult<GitPackMessage>
+>();
+
+const vaultsScan = new ServerCaller<
+  AgentRPCRequestParams,
+  AgentRPCResponseResult<VaultsScanMessage>
+>();
+
+// No type used here, it will override type inference
+const clientManifest = {
+  echo,
+  nodesChainDataGet,
+  // NodeClaimsGet,
+  nodesClosestLocalNodesGet,
+  nodesCrossSignClaim,
+  nodesHolePunchMessageSend,
+  notificationsSend,
+  vaultsGitInfoGet,
+  vaultsGitPackGet,
+  vaultsScan,
+};
+
+export {
+  clientManifest,
+  echo,
+  nodesChainDataGet,
+  // NodeClaimsGet,
```

Comment:
Why is this commented out?

Reply:
This and the one below are just placeholder. It existed in the GRPC handlers so I just recreated it here. I'm not sure if it's needed or why it was commented out before. I could just remove it.

Reply:
I don't know what it's for or where it came from. I think it's been here for a while. I can remove it, if it's needed at any point we can just make a new handler.

Reply:
I'm going to remove it.

Reply:
Removed.